### PR TITLE
Varvara/feature/base endpointіs

### PR DESCRIPTION
**Pull request by varvarashcherbyna7**

### CODE REVIEW

File: naples/routes/store.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    """Returns the store"""
+
+    store: m.Store | None = db.scalar(sa.select(m.Store).where(m.Store.uuid == store_uuid))
+    if not store:
+        log(log.ERROR, "Store [%s] not found", store_uuid)
+        raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail="Store not found")
+    return store
+
+
+# TODO: need to implement this route (may be it must be without current_user)
```
Comment (Taraslysak) [inline naples/routes/store.py]: Access to this route should be allowed only to admin, and nobody else

File: naples/routes/store.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    current_user: m.User = Depends(get_current_user),
+):
+    query = sa.select(m.Store)
+    stores: Sequence[m.Store] = db.scalars(query).all()
+    return s.Stores(stores=cast(list, stores))
+
+
+@store_router.post("/", status_code=status.HTTP_201_CREATED, response_model=s.StoreOut)
+def create_stote(
+    job: s.StoreIn,
```
Comment (Taraslysak) [inline naples/routes/store.py:54]: job?

File: tasks/fill_db_locations.py [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+        # ['city', 'city_ascii', 'state_id', 'state_name', 'county_fips', 'county_name', 'lat', 'lng', 'population', 'density', 'source', 'incorporated', 'timezone', 'zips', 'id']
+
+        # colums what we need for db
+        # ['city', 'city_ascii', 'state_id', 'state_name', 'county_name']
+
+        CITY_NAME_INDEX = header.index("city")
+
+        # TODO: check if we need this field (it seams this field the same as city)
+        # ASCII (American Standard Code for Information Interchange)
+        # CITY_ASCII_INDEX = header.index("city_ascii")
```
Comment (Taraslysak) [inline tasks/fill_db_locations.py]: I don't think we need this

File: tasks/fill_db_locations.py [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+        ABBREVIATED_NAME_INDEX = header.index("state_id")
+        STATE_NAME_INDEX = header.index("state_name")
+        COUNTY_NAME_INDEX = header.index("county_name")
+
+        rows: List[List[str]] = []
+        for row in csvreader:
+            rows.append(row)
+        log(log.INFO, "Rows count: %s", len(rows))
+
+        file.close()
```
Comment (Taraslysak) [inline tasks/fill_db_locations.py]: I don't think we need to create this list. We can process row one-by-one inside session scope

File: tests/test_data.py [hunk 5]
```diff
@@ -4,11 +4,29 @@
 class TestUser(BaseModel):
     __test__ = False
 
+    id: int
     email: str
     password: str
 
 
+class TestStore(BaseModel):
```
Comment (Taraslysak) [inline tests/test_data.py]: Why do we need to create these test schemas? why can't we use those from real code?

**Review by Taraslysak — COMMENTED** [review]
Plz consider some refactoring

File: naples/routes/store.py (hunk 1 above)
Comment (varvarashcherbyna7) [reply naples/routes/store.py]: ok

File: naples/routes/store.py (hunk 2 above)
Comment (varvarashcherbyna7) [reply naples/routes/store.py:54]: fix this

**Pull request merged**